ras_party/views/share_survey_view.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import structlog
+from flask import Blueprint, request, make_response, jsonify, current_app
+from werkzeug.exceptions import BadRequest
+
+from ras_party.controllers import share_survey_controller
+from ras_party.controllers.business_controller import get_business_by_id
+from ras_party.controllers.validate import Validator, Exists
+from ras_party.views.account_view import auth
+
+share_survey_view = Blueprint('share_survey_view', __name__)
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+
+
+@share_survey_view.before_request
+@auth.login_required
+def before_share_survey_view():
+    pass
+
+
+@auth.get_password
+def get_pw(username):
+    config_username = current_app.config['SECURITY_USER_NAME']
+    config_password = current_app.config['SECURITY_USER_PASSWORD']
+    if username == config_username:
+        return config_password
+
+
+@share_survey_view.route('/share-survey-users-count', methods=['GET'])
+def share_survey_users():
+    """
+    Get total users count who are already enrolled and pending share survey against business id and survey id
+    """
+    payload = request.get_json()
```

Comment:
This feels wrong.  If I remember right, having json payloads in HTTP GET is bad practice.  Even though 'share survey' isn't an HTTP resource as such, it'd be better to have the url be `/share-survey-users-count?business_id=<id>&survey_id=<id>`

Reply:
yes you right ... this needs changing